packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -126,7 +126,8 @@ export abstract class FileDialog<T> extends AbstractDialog<T> {
     constructor(
         @inject(FileDialogProps) readonly props: FileDialogProps,
         @inject(FileDialogWidget) readonly widget: FileDialogWidget,
-        @inject(LocationListRendererFactory) readonly locationListFactory: LocationListRendererFactory
+        @inject(LocationListRendererFactory) readonly locationListFactory: LocationListRendererFactory,
+        @inject(FileDialogTreeFiltersRendererFactory) readonly treeFiltersFactory: FileDialogTreeFiltersRendererFactory
```

Comment:
Since this is a new injection, maybe use property injection instead of constructor injection? Same below.

Reply:
I've refactored this a bit to utilize property injection for everything except for `props` and moved the remaining initialization to a `postConstruct`. I've also updated this for `SaveFileDialog` and `OpenFileDialog`. Let me know what you think 👍